inc/namespace.php

```diff
@@ -35,8 +35,15 @@ function bootstrap( array $config ) {
 		add_action( 'template_redirect', __NAMESPACE__ . '\\send_xss_header' );
 	}
 
+	if ( $config['content-security-policy'] ?? null ) {
```

Comment:
isn't `?? null` redundant?

Reply:
It avoids the notice if the key isn't set :)

Reply:
👍 had a context-switching issue between plugin & module then!